util/get-toolchain.py

```diff
@@ -49,6 +68,15 @@ def get_installed_toolchain_info(install_path):
         buildinfo = {}
         with open(str(install_path / 'buildinfo.json'), 'r') as f:
             buildinfo = json.loads(f.read())
+
+        # This is a temporary fix because we forgot this information in the
+        # buildinfo.json for toolchain 20200602-1
```

Comment:
nit: Sadness. No way to just throw this into the bundle?

Reply:
Just add it to a rebuild? 20200602-1 has never been used.

Reply:
Yeah, I'm working on it